srcpkgs/mu/template

```diff
@@ -10,8 +11,8 @@ short_desc="Emacs-based tool for dealing with e-mail stored in the Maildir-forma
 maintainer="Benjamin Slade <slade@jnanam.net>"
 license="GPL-3.0-or-later"
 homepage="https://www.djcbsoftware.nl/code/mu/"
-distfiles="https://github.com/djcb/mu/archive/${version}.tar.gz"
-checksum=8a6afd9f22a21499610bc07a19ea2c4dbe21cc8a2ce7924569346a5ae4b46199
+distfiles="https://github.com/djcb/mu/archive/1,4.7.tar.gz"
```

Comment:
`1,4.7` annoys me.

Can we use this instead:

    https://github.com/djcb/mu/releases/download/1%2C4.7/mu-1.4.7.tar.xz

Oh, wait, there's a `,` in the link, at least we don't need to change `wrksrc`

Reply:
it doesn't build if I don't change the `wrksrc`